docassemble/ALDashboard/data/questions/install_assembly_line.yml

```diff
@@ -198,31 +218,111 @@ fields:
-  for it.
+  Assembly Line uses GitHub to let you:
+  
+  1. Get user feedback about each interview.
+  2. Display a "last updated" date on the "About" page of an interview.
+
+  First, in order to use GitHub you should create a dedicated GitHub account. You can use an account that already exists, but we recommend having an account that only handles Assembly Line functionality.
+  
+  The account must have permission make pull requests on your GitHub organization's repositories. You can read [GitHub's documentation about setting an account's permissions](https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-roles-for-an-organization).
   
-  This feature is used to add feedback to production interviews as well
-  as to display a "last updated" date on the about page.
+  Give us the information about that account and set up its permissions.
 fields:
+  - What is the username of the account: the_config['github issues']['username']
+    default: ${ the_config['github issues'].get('username', '') }
+    help: |
+      This will set the `config['github issues']['username']` and the `config['github readonly']['username']` values.
   - note: |
-      #### GitHub Feedback form
-  - Username: the_config['github issues']['username']
-  - Private token: the_config['github issues']['token']
-  - Default Github organization: the_config['github issues']['default repository owner']
-  - Allowed repository owners (one per line): allowed_github_repo_owners
+      Now you need to authorize your docassemble server to use that account to make issues. You can do that by creating a personal access token (PAT):
+
+      1. Follow directions to start making a personal access token (PAT) for the account on GitHub. You can find the directions in a section at [this GitHub documentation page](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token).
+      1. Set the expiration date to "No expiration".
+      1. Tap the checkbox for “repo” permissions.
+      1. Finish creating the personal access token.
+      1. Copy the token and paste it below
+  - Personal access token: the_config['github issues']['token']
+    default: ${ the_config['github issues'].get('token', '') }
+    help: |
+      This will set the `config['github issues']['token']` and the `config['github readonly']['token']` values.
+---
+id: github owners list
+continue button field: github_owners_list
+question: |
+  Avoid GitHub issue spamming
+subquestion: |
+  To make sure that no one can use the feedback forms to spam just any GitHub repository with issues, you must limit which repositories your feedback forms can access. You can do this by limiting your issues to repositories owned by specific users or organizations.
+      
+  An interview can name its own repository's owner, but you must make sure that the owner is an owner that you have approved.
```

Comment:
I'd remove this line. Interview specific naming of repo owners is a fairly complex feature for this set-up interview. If we want to have a GithubFeedback specific docs page (either with the other docs, or just the Readme in the repo), this information would go there and we could just add a link at the end of the setup interview saying "If you want to learn more about the GitHub Feedback feature, go here". 
```suggestion
```

Reply:
How do we explain why they need to make a list?

Reply:
IMO that's explained in the previous line: "You can do this by limiting your issues to repositories owned by specific *users* or *organizations*." The plurals explains it well enough IMO.